src/test/java/igrad/logic/commands/DeleteCommandTest.java

```diff
@@ -0,0 +1,108 @@
+package igrad.logic.commands;
+
+import static igrad.logic.commands.CommandTestUtil.assertCommandFailure;
+import static igrad.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static igrad.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+import igrad.commons.core.Messages;
+import igrad.commons.core.index.Index;
+import igrad.model.Model;
+import igrad.model.ModelManager;
+import igrad.model.UserPrefs;
+import igrad.model.module.Module;
+import igrad.testutil.TypicalIndexes;
+import igrad.testutil.TypicalPersons;
+
+/**
+ * Contains integration tests (interaction with the Model, UndoCommand and RedoCommand) and unit tests for
+ * {@code DeleteCommand}.
+ */
+public class DeleteCommandTest {
+
+    private Model model = new ModelManager(TypicalPersons.getTypicalCourseBook(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        Module moduleToDelete = model.getFilteredModuleList().get(TypicalIndexes.INDEX_FIRST_PERSON.getZeroBased());
```

Comment:
if you had retained the static import (above), this line of code should look this;
'Module moduleToDelete = model.getFilteredModuleList().get(INDEX_FIRST_PERSON.getZeroBased());'

(please refer to above comment)